### PR TITLE
Prepare 5.16.0

### LINKED ISSUE #3815: Call Activity Propagation data marker does not update
### Describe the bug

Recording 2023-08-30 at 11 14 06 (1)

### Steps to reproduce

1. Create a Call activity in C8
2. Toggle on Input or Output propagation
3. Toggle off again
4. The Group data marker is still there

### Expected behavior

The data marker is removed when I toggle propagation off.

## Camunda Modeler system information
 * Version: 5.15.0-dev
 * Operating System: Linux x86_64 amd64
 * Execution Platform: BPMN - Camunda Platform 8

https://github.com/camunda/camunda-modeler/pull/3814

### Additional context

The data marker is updated when another property in the `zeebe:calledElement` is updated, e.g. other Propagation or called element ID

### PR DESCRIPTION
**Pull request by nikku**
This prepares the [5.16.0 release](https://github.com/camunda/camunda-modeler/issues/3846).

---

Proposed update of what's new communication:

image

----

Closes #3815

### CODE REVIEW

**Review by barmac — APPROVED** [review]
LGTM! Grammarly is also super happy ;)

**Comment by nikku** [comment]
Thanks for the additional grammar check :checkered_flag:.

**Comment by barmac** [comment]
I've added @christian-konrad and @YanaSegal to review so that they can share their feedback too (cf. release issue).

**Comment by RomanKostka** [comment]
Excellent, thanks @nikku & @barmac!

One thing I would like to propose is changing the form section to:

**Forms improvements**
Added group component and the support of nested data ( form keys )

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 1]
```diff
@@ -44,20 +44,20 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Validation improvements</h4>
-          Errors shown in the Problems panel now link to relevant documentation making it easier to understand the error and how to fix it.
+          <h4>Forms improvements</h4>
+          Forms ship with groups and support nested form keys.
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:48]:
```suggestion
          Added a group component and the support for nested data (form keys)
```

**Comment by nikku** [comment]
Thanks @RomanKostka. Incorporated your suggestion via https://github.com/camunda/camunda-modeler/pull/3897/commits/a4c9b0eec0bc4bdcf9dd3506875da18e68720e36.

**Pull request merged**